src/frontend/src/flows/dappsExplorer/teaser.ts

```diff
@@ -0,0 +1,85 @@
+import { html, TemplateResult } from "lit-html";
+import { dapps } from "./dapps";
+
+/** A teaser for the dapps explorer, acting as a button */
+export const dappsTeaser = ({
+  click,
+}: {
+  click: () => void;
+}): TemplateResult => {
+  return html`<article class="c-card c-card--narrow">
+    <h3 class="t-title t-title--discrete" role="presentation">
+      Dapps explorer
+    </h3>
+    <h2 class="t-title">Explore dapps</h2>
+    <button
+      class="c-click-area"
+      @click="${() => click()}"
+      aria-label="Show list of dapps"
+    >
+      <figure class="c-card__teaser c-marquee c-marquee--clickable">
+        ${marqueeList()}
+      </figure>
+    </button>
+  </article>`;
+};
+
+const marqueeList = (): TemplateResult => {
+  const itemsPerRow = 5;
```

Comment:
should these constants be declared somewhere else, like in a config or constant dedicated file?

Reply:
they're only ever used here so I'd rather keep it as simple as possible for now